src/clarity-angular/datagrid/datagrid-action-overflow.spec.ts

```diff
@@ -80,11 +73,18 @@ export default function(): void {
             context.detectChanges();
             expect(context.clarityDirective.open).toBe(true);
 
-            // should close when inside the overflow menu is clicked
-            let actionItem: HTMLElement = context.clarityElement.querySelector(".datagrid-action-overflow");
+            // should not close when the overflow menu container is clicked
+            let actionOverflowMenu: HTMLElement = context.clarityElement.querySelector(".datagrid-action-overflow");
+            actionOverflowMenu.click();
+            context.detectChanges();
+            expect(context.clarityDirective.open).toBe(true);
+
+            // should close when a menu item is clicked
+            let actionItem: HTMLElement = context.clarityElement.querySelector(".action-item");
             actionItem.click();
             context.detectChanges();
             expect(context.clarityDirective.open).toBe(false);
+
         });
```

Comment:
This test is starting to do a bit too much for my taste. The first part (closed initially, opens on ellipsis clicked) is already tested separately, so it can be removed. Then a test that has "XXX but YYY" clearly tests 2 things, so it could easily be split in 2 atomic tests, "XXX" and "YYY".

Still, I'm not going to block this PR for this, just a remark if you have time.